editor/src/clj/editor/types.clj

```diff
@@ -190,6 +190,8 @@
    width    :- Int32
    height   :- Int32
    sprite-trim-mode :- sprite-trim-modes]
+  {(s/optional-key :digest-ignored/error-node-id) Long}
+
```

Comment:
Schema change to allow us to `assoc` additional keys into the `Image` record. Since this will end up inside build target `:user-data`, we need to ignore it when generating the sha1-hash from the build target. Node ids can differ between runs of the editor, and the sha1-hash is used to cache build target results on disk to avoid rebuilds.